CMakeLists.txt

```diff
@@ -442,6 +448,13 @@ if (MRDOCS_INSTALL)
     #-------------------------------------------------
     # share
     #-------------------------------------------------
+    install(DIRECTORY ${LIBCXX_DIR}
+            DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/mrdocs/libcxx
+            FILES_MATCHING PATTERN "*")
+    install(DIRECTORY ${CLANG_STDLIB_INCLUDE_DIR}
+            DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/mrdocs/clang
```

Comment:
`clang` is bad naming unless this directory includes, well... the clang compiler.

Reply:
Also `STDLIB_INCLUDE_DIR` is confusing. Isn't `LIBCXX_DIR` already an implementation of the standard library? What's in `STDLIB_INCLUDE_DIR`? How does it differ from `LIBCXX_DIR`?